### DISCUSSION TITLE
Incremental Static Regeneration on Netlify

### DISCUSSION THREAD

**Discussion by jvink** [vercel/next.js#17312] [Help] [answered]
My data source is a github repo that consists of data and gets updated every 24 hours or so. My function looks like this:

```
export async function getStaticProps() {
  const result = await fetch(...);
  const data = await result.json();

  return {
    revalidate: 60 * 60 * 2,
    props: {
      data,
    },
  };
}

```
The problem is that the site does not get updated, even though the data from the api changed. The site is on a git repo that I authorised to Netlify in order to build the site on every deploy. But as stated in the docs, I should not have to build every time to have this dynamic content be updated. Is it a limitation of using Netlify or am I doing something completely wrong?

**Comment by timneutkens** [comment, marked as answer]
Netlify only supports `next export` which does not work with Incremental Static Generation as it exports 100% static HTML. You can use vercel.com or any other provider that supports hosting Node.js applications.

**Reply by jvink** [reply]
Thanks! [Deployed](https://covid19-data.nl/) in on Vercel now. Works like a charm. Cheers!

**Comment by OnrCan** [comment]
Update: Netlify now supports ISR and many other Next.js features with a plugin.

[Announcing one-click install Next.js Build Plugin on Netlify  ](https://www.netlify.com/blog/2020/12/07/announcing-one-click-install-next.js-build-plugin-on-netlify/)
